### ISSUE TITLE
docs: Continuous Delivery page navbar is hard to parse

### ISSUE THREAD

**Issue by ahmetb** [GoogleContainerTools/skaffold#6513]
The code fonts in the section titles are making the right part quite hard to read and understand. 

![image](https://user-images.githubusercontent.com/159209/130850379-89fa6add-8bca-471d-941a-c5a09707a822.png)

**Comment by verbanicm** [comment]
Happy to take this, I feel like removing the markdown code block from the header will help this.

**Comment by ahmetb** [comment]
@verbanicm yeah I think alternative titles, or figuring out if page's rightside navbar levels can be reduced per-page in Docsy docs might be to go.
